packages/build/webpack/constants.js

```diff
@@ -39,6 +39,17 @@ if (process.env.npm_package_dependencies__labkey_workflow) {
 
 const watchPort = process.env.WATCH_PORT || 3001;
 
+const minifyTemplateOptions = {
```

Comment:
I would add a comment above this describing how this set of options was chosen.

```js
// These minification options are a re-declaration of the default minification options 
// for the HtmlWebpackPlugin with the addition of `caseSensitive` because LabKey's 
// view templates can contain case-sensitive elements (e.g. `<permissionClasses>`). 
// For more information see https://github.com/jantimon/html-webpack-plugin#minification.
const minifyTemplateOptions = {
    caseSensitive: true,
    collapseWhitespace: process.env.NODE_ENV === "production",
    keepClosingSlash: true,
    removeComments: true,
    removeRedundantAttributes: true,
    removeScriptTypeAttributes: true,
    removeStyleLinkTypeAttributes: true,
    useShortDoctype: true
}
```